Header Header
float64 Latitude
float64 Longitude
float64 Altitude
float64 utm_easting
float64 utm_northing
float64 zone
string letter
float64 Fix_quality
